src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.MaxResponseHeadersLength.cs

```diff
@@ -125,7 +131,11 @@ public async Task ThresholdExceeded_ThrowsException(int? maxResponseHeadersLengt
                     headers.Add(new HttpHeaderData($"Custom-{i}", new string('a', 480)));
                 }
 
-                await server.HandleRequestAsync(headers: headers);
+                try
+                {
+                    await server.HandleRequestAsync(headers: headers).WaitAsync(TestHelper.PassingTestTimeout);
+                }
+                catch { }
```

Comment:
```suggestion
                catch (Exception ex)
                {
                    _output.WriteLine($"Ignored exception:{Environment.NewLine}{ex}");
                }
```
It comes sometimes handy when investigating and we have this in every discarded test exception in Http AFAIK.

Reply:
I don't think I ever added these before, but I can see it being useful